cli/cmds/build.js

```diff
@@ -30,18 +31,32 @@ async function build() {
     await androidSdkTools.installBuildTools();
   }
 
+  const twaManifest = await TwaManifest.fromFile('./twa-manifest.json');
   prompt.message = colors.green('[llama-pack-build]');
   prompt.delimiter = ' ';
   prompt.start();
 
+  const schema = {
```

Comment:
As a larger point (not for this CL), requiring the passwords to be entered as prompts makes it tricky to use this tool as part of a larger script. Filed #30 for this.

Reply:
+1